changelog.readme

```diff
@@ -1,3 +1,6 @@
+## 1.2.2
```

Comment:
Since it's a feature: then we could bump to 1.3.0

Reply:
Good point, changed to 1.3.0